### ISSUE TITLE
Calculated fall effect buggy

### ISSUE THREAD

**Issue by Mahadbaloch** [mariumbacchus/Soulslike-Weaponry#49]
It does not work often. It feels like rng. Sometimes I take fall damage even though i have the effect, sometimes it works, and sometimes i dont take fall damage for a few seconds, and then suddenly I get the damage. Either it really is broken or it is very sensitive and the slightest lag causes it to not work.

**Comment by mariumbacchus** [comment]
If you lag often, then there is nothing I can do. It is connected to when you take damage on the server side, so if you are out of sync for a few seconds, then I see why it happens. I'll look into it and see if something can be done.

**Comment by Mahadbaloch** [comment]
I just tested a bit and found something weird. It even happens on single player when there is no lag, and while having the effect I have this weird dirt particle effect constantly spamming under me. I also found that when I use the golden spear to shoot up, and then I fall with the effect, If i stay still, I will take damage, but if I immediately move away from where I fell, i dont die. Kinda weird
